lib/gc/ExecutionEngine/CPURuntime/MemoryPool.cpp

```diff
@@ -230,13 +230,6 @@ void FILOMemoryPool::release() {
   current = nullptr;
 }
 
-void FILOMemoryPool::clear() {
-  for (auto cur = current; cur; cur = cur->prev) {
-    cur->allocated = sizeof(MemoryBlock);
-  }
-  current = buffers;
-}
-
```

Comment:
is this modification related to the issue？

Reply:
No. This is an extra modification, remove the useless function.